.changeset/purple-dryers-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus-editor": patch
+---
+
+Fix templates in the ItemEditor
```

Comment:
If the logic wasn't working it makes me think we don't have proper test coverage for the feature. Could we add that?